OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -70,6 +70,8 @@ function locate_java() {
 
     JAVA_HOME="$(dirname "$(dirname "$(readlink -f "${java}")")")"
     export JAVA_HOME
+    LIB_JVM_PATH="$(dirname "$(find $JAVA_HOME -name libjvm.so)" )"
```

Comment:
LIB_JVM_PATH must be `local`. A better name for this variable would be `JVM_LIBRARY_PATH`

Reply:
Done